Scoop/Controller/Ride Posting/PostRideSummaryViewController.swift

```diff
@@ -21,6 +21,10 @@ class PostRideSummaryViewController: UIViewController {
     
     private var ride: Ride!
     
+    // MARK: Networking
+    private let networkManager = NetworkManager()
```

Comment:
NetworkManger should be a singleton, only one instances `shared` should be used universally throughout the entire codebase

Reply:
On this thread, NetworkManager should have a `private init() {}` to prevent instances like this from being made outside of the shared instance